include/PCH.h

```diff
@@ -31,6 +31,8 @@ void* operator new[](size_t size, size_t alignment, size_t alignmentOffset, cons
 #define WIN32_LEAN_AND_MEAN
 #include <Windows.h>
 
+#define TRACY_ENABLE
```

Comment:
This should be a cmake option that is disabled by default. 

```cmake
target_compile_definitions(
	"${PROJECT_NAME}"
	PUBLIC
	"$<$<BOOL:${TRACY_ENABLE}>:TRACY_ENABLE=1>"
)

```

Reply:
PR please